managers/int.py

```diff
@@ -839,7 +839,7 @@ async def _send_flows(
                     content={
                         "dpid": dpid,
                         "force": True,
-                        "flow_dict": {"flows": flows},
+                        "flow_dict": {"flows": flows_batch},
```

Comment:
I didn't add more unit test since the batching will be replaced on https://github.com/kytos-ng/telemetry_int/issues/47, so it would only create extra test that would need to be nuked soon.